test/python/result/test_sampled_expval.py

```diff
@@ -82,17 +81,12 @@ def test_same(self):
         exp2 = sampled_expectation_value(counts, Pauli(oper))
         self.assertAlmostEqual(exp2, ans)
 
-        with self.assertWarns(DeprecationWarning):
-            exp3 = sampled_expectation_value(counts, PauliOp(Pauli(oper)))
-        self.assertAlmostEqual(exp3, ans)
-
         spo = SparsePauliOp([oper], coeffs=[1])
-        with self.assertWarns(DeprecationWarning):
-            exp4 = sampled_expectation_value(counts, PauliSumOp(spo, coeff=2))
-        self.assertAlmostEqual(exp4, 2 * ans)
+        exp3 = sampled_expectation_value(counts, spo)
+        self.assertAlmostEqual(exp3, ans)
 
-        exp5 = sampled_expectation_value(counts, SparsePauliOp.from_list([[oper, 1]]))
-        self.assertAlmostEqual(exp5, ans)
+        exp4 = sampled_expectation_value(counts, SparsePauliOp.from_list([[oper, 1]]))
+        self.assertAlmostEqual(exp4, ans)
 
```

Comment:
```suggestion
```

this is the same test as `exp3` above now, just spelled differently

Reply:
oh, I thought I had removed that repetition, thx!